utils/helper.js

```diff
@@ -18,4 +18,9 @@ const getPagination = (page, size) => {
   };
 };
 
-module.exports = { getPagination, getPagingData };
+const getDueDate = () => {
```

Comment:
Maybe later, we can make this dynamic such that it will depend on admin discretion or maybe the book type/the student